apps/session-manager/src/utils/responses.ts

```diff
@@ -70,6 +73,19 @@ export const withCatchAsInternalError = <T>(
     return ResponseErrorInternal(`${message} [${_}]`);
   });
 
+export const withLeftAsInternalError = <T>(
```

Comment:
as my other comment says, we can remove this if the getProfile becomes a `RTE`

Reply:
Fixed in 374cc39